src/simsopt/configs/zoo.py

```diff
@@ -187,3 +195,44 @@ def get_w7x_data(Nt_coils=48, Nt_ma=10, ppp=2):
     ma.zs[:] = sZ[0:Nt_ma]
     ma.x = ma.get_dofs()
     return (curves, currents, ma)
+
+
+@SimsoptRequires(requests is not None, "You need to install the requests library to use this function. Run 'pip install requests'")
+def get_QUASR_data(ID, return_style='default'): 
+    """
+    Download a configuration from the QUASAR database.
+
+    Args:
+        ID: the ID of the configuration to download.  The database is navigatable at https://quasr.flatironinstitute.org/
+            Alternatively, you can download the latest full set of devices from https://zenodo.org/doi/10.5281/zenodo.10050655
+        
+        return_style: 'default' or 'json'. If 'default', the function will return the curves, currents and magnetic axis
```

Comment:
The name `"json"` isn't an intuitive description of what the function returns. Maybe `"surfaces"`?

Reply:
Agreed--it also looks like "default" is only returning one half-period? Am I reading that right?